### DISCUSSION TITLE
multiple dynamic routes

### DISCUSSION THREAD

**Discussion by will-i-code420** [nuxt/nuxt#19354] [Questions] [answered]
i am trying to create several dynamic routes in the same main route, is this even possible?  It always defaults to the first dynamic route in the folder.  I'm trying to setup something like below

`-| pages/`
`---| index.vue`
`---| admin/`
`-----| [id].vue`
`-----| [product].vue`
`-----| index.vue`

the id would be like a profile edit and product would be for editing products

**Comment by AloisSeckar** [comment, marked as answer]
You can have partial matches in dynamic routes, so something like this would do:

```
-| pages/
---| index.vue
---| admin/
-----| user-[id].vue
-----| product-[id].vue
-----| index.vue
```

**Reply by will-i-code420** [reply]
that is what i ended up doing, just had to change several route names.  But it is working as it should now
